src/services/submissionService.ts

```diff
@@ -101,7 +155,31 @@ export const mapAndValidateSpokenLanguages = (spokenLanguages: SpokenLanguageInp
     return validatedLanguages.map(gqlSpokenLanguageToDbSpokenLanguage)
 }
 
-const mapDbEntityTogqlEntity = (dbEntity: DocumentData) : Submission => {
+export const mapGqlFiltersToDbFilters = (filters: gqlType.SubmissionFilters):
+    dbSchema.SubmissionFilters => {
+    const mappedLanguages = filters.spokenLanguages?.map(lang => lang ? {
+        iso639_3: lang.iso639_3 || '',
```

Comment:
I think you mean't to use the `??` null coalescing operator for these
```suggestion
        iso639_3: lang.iso639_3 ?? '',
```

Reply:
@ermish I didn't know about this `??` null coalescing operator. Now I know 😁 Thank you 🙏